merlin/config/broker.py

```diff
@@ -139,7 +139,9 @@ def get_redissock_connection(config_path, include_password):
     return REDISSOCK_CONNECTION.format(**redis_config)
 
 
-def get_redis_connection(config_path, include_password, ssl=False):
+# flake8 complains this function is too complex, we don't gain much nesting any of this as a separate function,
```

Comment:
a lot ofd the complexity could be reduced if the CONFIG.broker object  had default values set there